### ISSUE TITLE
lint: add commented-out code detector

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#337]
Detects commented-out code inside function bodies.

**Before:**
```go
// fmt.Println("Debugging hard")
foo(1, 2)
```
**After:**
```go
foo(1, 2)
```

Not sure this can be done without both false positives and false negatives, but still would be useful if it can detect at least some instances. Can help during big and old (legacy) codebase cleanup.

**Comment by Quasilyte** [comment]
Seems like it hard to be low at false and positive and negatives at once here.
I'll post some good matches here.
```
cmd/internal/obj/x86/asm6.go:2034:13
	if false { /* debug['a'] > 1 */
```

**Comment by NexoMichael** [comment]
Maybe as "first" step it will be good to check `fmt.Print()` statements in comments? From my experience it always was some kind of "debug" code that was incidentally added.

**Comment by Quasilyte** [comment]
@NexoMichael , yeah, could be a good first step, but it already catches a few more ugly cases.
